PrestaShopCorp/psgdpr
# Add a join to load only existing modules

|Questions| Answers|
|----------|-----------------------------------------|
|Description? | Modules are added in the table psgdpr_consent but are never removed from this table even when uninstalled, and still appear as ghost modules in the BO. This additional join in SQL query fixes this issue.
|Type? | bug fix
|Category?| BO
|BC breaks?| no
|Deprecations?| no
|How to test?| See bellow

1) Install psgdpr module
2) Configure psgdpr module
3) Uninstall a module previously configured in psgdpr module
4) See in the second and third tabs of psgdpr module configuration page, there is no more ghost modules (without a name, icon etc)
